taker-frontend/src/App.tsx

```diff
@@ -53,8 +53,11 @@ export const App = () => {
     const parcelSize = parseOptionalNumber(order?.parcel_size) || 0;
     const liquidationPrice = parseOptionalNumber(order?.liquidation_price);
     const marginPerParcel = order?.margin_per_parcel || 0;
-    const openingFee = order?.opening_fee || 0;
+    const leverage = order?.leverage || 0;
```

Comment:
If you default this to 0 now, you should change the `Props` type of `Trade` to no longer accept an optional `leverage` to get more type safety.